.gitignore

```diff
@@ -46,6 +46,7 @@ pip-delete-this-directory.txt
 # Unit test / coverage reports
 htmlcov/
 .tox/
```

Comment:
No, but I would prefer not remove because users might have `.tox` directories lying around.